From 0f7f09c6ad63f0955f508caa7e2d9ae6399091c3 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Fri, 24 Jul 2020 09:48:43 +0800
Subject: [PATCH] Test case for Library - CpuFlushTlb.c file

Description: Change the Flush address in the MdePkg/Library/BaseCpuLib/Ia32/CpuFlushTlb.c.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdePkg/Library/BaseCpuLib/Ia32/CpuFlushTlb.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/MdePkg/Library/BaseCpuLib/Ia32/CpuFlushTlb.c b/MdePkg/Library/BaseCpuLib/Ia32/CpuFlushTlb.c
index 89341c2f03..0d6f7a54a8 100644
--- a/MdePkg/Library/BaseCpuLib/Ia32/CpuFlushTlb.c
+++ b/MdePkg/Library/BaseCpuLib/Ia32/CpuFlushTlb.c
@@ -21,8 +21,8 @@ CpuFlushTlb (
   )
 {
   _asm {
-    mov     eax, cr3
-    mov     cr3, eax
+    mov     eax, cr2
+    mov     cr2, eax
   }
 }
 
-- 
2.27.0.windows.1

